### DISCUSSION TITLE
Is OPENVPN_REGIONS enough or do I need SERVER_NAMES?

### DISCUSSION THREAD

**Discussion by sunbeam60** [qdm12/gluetun#937] [Q&A] [answered]
Last night watchtower updated to gluetun:latest (d4bff13d9b29) and with it, a new servers.json file which refreshed some PIA servers.

This caused Gluetun to get stuck in a restart loop as some of my SERVER_NAMES were invalid.

I am keen to avoid this issue in the future - I don't care specifically which servers are connected to, but I do care about which country.

Is it enough to specify OPENVPN_REGIONS=Denmark,Sweden,Germany, say? Gluetun seems to pick a random server (say brussels408) when I comment out the SERVER_NAMES not respecting OPENVPN_REGIONS.

If SERVER_NAMES are required, what is OPENVPN_REGIONS actually used for?

**Comment by qdm12** [comment, marked as answer]
`OPENVPN_REGIONS` doesn't exist, where did you see it?

It's `SERVER_REGIONS` instead. And then gluetun will pick at random from the filtered servers. Using server names or hostnames is indeed risky if one of them disappears.

**Comment by sunbeam60** [comment]
Ha! That would explain it and I wish I could explain where OPENVPN_REGIONS came from. Possibly when I set things up originally, where I used a guide for a generic Open VPN container. Most likely my dysfunctional mind.

Changed to SERVER_REGIONS and all works as expected. Please forgive this waste of your time.

**Reply by qdm12** [reply]
Ah no worry! 👍
